debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorTask.java

```diff
@@ -105,7 +106,7 @@ public String version() {
                     schemaNameAdjuster);
 
             ChangeEventSourceCoordinator<MongoDbPartition, MongoDbOffsetContext> coordinator = new ChangeEventSourceCoordinator<>(
-                    Collections.singletonMap(new MongoDbPartition(), previousOffset),
+                    new Offsets<>(Collections.singletonMap(new MongoDbPartition(), previousOffset)),
```

Comment:
Let's add a constructor for the single-partition use case:

```
new Offsets<>(new MongoDbPartition(), previousOffset)
```

Reply:
This code is temporary. Once the MongoDB connector is refactored, this constructor won't be relevant.

Normally, the partition provider provides a set of partitions and the loader produces a map. The MongoDB connector has two sets of partitions/offsets: `MongoDbPartition`/`MongoDbOffsetContext` (the "fake" ones to satisfy the framework API) and `ReplicaSetPartition`/`ReplicaSetOffsetContext` (managed internally by the connector).

Reply:
Ah, gotcha. I was under the impression it'd be usedd by the single-partition connectors, too. If that's not the case, agreed, no need to change things here.